data_storage/export_format.md

```diff
@@ -1,22 +1,28 @@
 Export Format
 -------------
 
-- **5.1.1** ![](/badge/req.png) A Toxcore data file saved in the root of the client's
-  working directory, with the file name `<profile_name>.tox`
+- **5.1.1** ![](/badge/req.png) The client will export files relative to a folder called
+  the export folder, which must be in a location accessible to the user.
```

Comment:
What does "accessible" mean? Does it mean writeable by the user running the process or does it mean easily discoverable by the user operating the machine or maybe something different again?